testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode/multinode-worker-jvm.config

```diff
@@ -1,8 +1,5 @@
 -server
 -Xmx2G
--XX:-UseBiasedLocking
-# Disable G1 to enable UseGCOverheadLimit (JDK-8212084) and reduce JVM's memory footprint
--XX:-UseG1GC
```

Comment:
Should `UseGCOverheadLimit` be removed now? or does Java 17 support UseGCOverheadLimit with G1?